.github/workflows/build-and-push.yml

```diff
@@ -0,0 +1,52 @@
+name: Publish image to GitHub Container Registry
+
+on:
+  push:
+    branches: ["main"]
+
+jobs:
+  publish-image:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout project
+        uses: actions/checkout@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Docker Metadata action
```

Comment:
Am I understanding this right that this tags the image based on the Github version tags we create manually on release? 

If so that seems sufficient for now rather than implementing automated versioning, although automating the semver tag creation is also an option.

Reply:
I merely ripped off this metadata config from poktroll repo. I haven't been able to see that your statement is what it does (like you, I assume that's how it works). I also think automating semver is an good option, so I would say whether we go for one or the other, we should be just fine.